src/ResourceManagement/MachineLearning/Microsoft.Azure.Management.MachineLearning/generate.cmd

```diff
@@ -0,0 +1,17 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+set autoRestVersion=0.16.0-Nightly20160501
+if  "%1" == "" (
+    set specFile="https://raw.githubusercontent.com/tudordid/azure-rest-api-specs/master/arm-machinelearning/2016-05-01-preview/swagger/webservices.json"
```

Comment:
Correct, I forgot to update this after we have pushed the swagger spec in the main repo. Will fix.